src/main/java/edu/umich/ctools/esb/utils/WAPI.java

```diff
@@ -80,33 +126,105 @@ public void setApiPrefix(String apiPrefix) {
 		this.apiPrefix = apiPrefix;
 	}
 	
-	//a token must be created at the time of construction
+	//A token must be created at the time of construction
 	//this token will allow use of the ESB APIs
 	public String buildRenewal(String key, String secret) {
 		String b64 = base64KeySecret(key, secret);
 		b64 = "Basic " + b64; 
 		return b64;
 	}
 
-	//esb calls require base 64 strings for authroization
+	//esb calls require base 64 strings for authorization
 	public String base64KeySecret(String key, String secret) {
 		String keySecret = key + ":" + secret;
 		byte[] binaryData = keySecret.getBytes();
 		keySecret = Base64.encodeBase64String(binaryData);
 		return keySecret;
 	}
 
-	//perform ESB request. If there is an exception, return exception result response.
+	/*********************************/
+	// Compatibility wrappers for old doRequest that defaults to doGetRequest
 	public WAPIResultWrapper doRequest(String request){
-		M_log.info("doRequest: " + request);
+		return doGetRequest(request,null);
+	}
+	public WAPIResultWrapper doRequest(String request,HashMap<String,String> headers){
```

Comment:
Can you document what all the header values that is expected. I see the for CCM is not sending any header and SPE looks like using it. I am having hard time understanding what is being sent as part of headers